src/app/two-factor/two-factor.module.ts

```diff
@@ -1,29 +1,30 @@
-import { NgModule } from '@angular/core'
 import { CommonModule } from '@angular/common'
-import { TwoFactorComponent } from './pages/two-factor/two-factor.component'
-import { TwoFactorRoutingModule } from './two-factor-routing.module'
-import { MatProgressBarModule } from '@angular/material/progress-bar'
+import { NgModule } from '@angular/core'
+import { MatButtonModule } from '@angular/material/button'
 import { MatCardModule } from '@angular/material/card'
-import { SignInModule } from '../sign-in/sign-in.module'
-import { MdePopoverModule } from '@material-extended/mde'
+import { MatFormFieldModule } from '@angular/material/form-field'
 import { MatIconModule } from '@angular/material/icon'
-import { MatButtonModule } from '@angular/material/button'
+import { MatProgressBarModule } from '@angular/material/progress-bar'
+import { MdePopoverModule } from '@material-extended/mde'
+
 import { A11yLinkModule } from '../cdk/a11y-link/a11y-link.module'
-import { MatFormFieldModule } from '@angular/material/form-field'
+import { TwoFactorAuthenticationFormModule } from '../cdk/two-factor-authentication-form/two-factor-authentication-form.module'
+import { TwoFactorComponent } from './pages/two-factor/two-factor.component'
+import { TwoFactorRoutingModule } from './two-factor-routing.module'
 
 @NgModule({
   declarations: [TwoFactorComponent],
   imports: [
     CommonModule,
     TwoFactorRoutingModule,
-    SignInModule,
```

Comment:
I did move the `old` two-factor component into its own module `TwoFactorAuthenticationFormModule` so we don't have to import the huge `SignInModule` to consume it.